packages/storybook/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "outDir": "build/",
     "rootDirs": ["src"],
     "types": ["node", "requestidlecallback"],
-    "sourceMap": true
```

Comment:
DataGrid doesn't work in storybook in PR's netlify deployment: https://deploy-preview-3965--material-ui-x.netlify.app/storybook
<img width="1015" alt="Screenshot 2022-02-22 at 11 28 09" src="https://user-images.githubusercontent.com/13808724/155117182-49f2cdeb-c4a1-44fa-8cb4-adfef4c46e53.png">

Reply:
RELEASE_INFO wasn't replaced in storybook:
<img width="503" alt="Screenshot 2022-02-22 at 11 59 06" src="https://user-images.githubusercontent.com/13808724/155118903-ab3aec10-ba53-4de0-bfcc-2325def3e67b.png">

---

It's not an issue in packages available on CodeSandbox CI:

<img width="630" alt="Screenshot 2022-02-22 at 12 00 00" src="https://user-images.githubusercontent.com/13808724/155119044-2213d4ab-1e6a-47f6-ace6-bb7f35d6efca.png">

Reply:
Probably a wrong config on `babel.config.js`

Reply:
I updated the Storybook config, it should solve the problem
I'll test once deployed

Reply:
Fixed